TLM/TLM/U/HelperExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace TrafficManager.U {
+    using ColossalFramework.UI;
+    using UnityEngine;
+
+    public static class HelperExtensions {
+        /// <summary>
+        /// Get rectangle of a control in GUI screen space.
+        /// GUI screen space is used to draw overlay textures in overlay helpers.
+        /// </summary>
+        public static Rect GetScreenRectInGuiSpace(this UIComponent ctrl) {
```

Comment:
I see no calls to this method and I have no way of testing this.
is this redundant or is it basis for future code?

Reply:
Is used in future code, that is not merged in this PR. This PR merges entire new U directory so some pieces are here but not used yet.